DEVDOCS-4972:[update] Update custom-templates.mdx (#1955)
bigcommerce/dev-docs: docs/stencil-docs/storefront-customization/custom-templates.mdx

* Product
* Page

> This section explains how to author a custom template, map it to a URL, upload it to BigCommerce, and apply it to a storefront as a theme using Stencil CLI. Mapping a template to a URL is specifically for local testing.

Reviewer: ```suggestion
This section explains how to: author a custom template, map it to a URL, upload it to BigCommerce, and apply it to a storefront as a theme using Stencil CLI. Mapping a template to a URL is specifically for local testing.
```

Author: @bc-tgomez I would like to keep this as is because the text includes a sentence and bullet points.

## Authoring a custom template

> If you are a theme developer, your first step is to create a custom template page. 

Reviewer: ```suggestion
If you're a theme developer, your first step is to create a custom template page. 
```

Author: @bc-tgomez I think the contractions are needed.
